source/extensions/compression/zstd/compressor/config.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include "envoy/compression/compressor/factory.h"
+#include "envoy/extensions/compression/zstd/compressor/v3/zstd.pb.h"
+#include "envoy/extensions/compression/zstd/compressor/v3/zstd.pb.validate.h"
+
+#include "source/common/http/headers.h"
+#include "source/extensions/compression/zstd/compressor/zstd_compressor_impl.h"
+#include "source/extensions/compression/common/compressor/factory_base.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Compression {
+namespace Zstd {
+namespace Compressor {
+
+namespace {
+
+const std::string& zstdStatsPrefix() { CONSTRUCT_ON_FIRST_USE(std::string, "zstd."); }
+const std::string& zstdExtensionName() {
+  CONSTRUCT_ON_FIRST_USE(std::string, "envoy.compression.zstd.compressor");
+}
```

Comment:
I think these constants should be updated to `constexpr absl::string_view`.

Reply:
Hi @zhxie , it is true `absl::string_view` is better than `std::string`, but here is relate to [CompressorLibraryFactoryBase](https://github.com/envoyproxy/envoy/blob/main/source/extensions/compression/common/compressor/factory_base.h#L32), should we update it when we upgrade all compressor module?

Reply:
Since we have already used `const string&`, then it is Ok to not touch it.